p2p/transport/webrtc/util.go

```diff
@@ -0,0 +1,76 @@
+package libp2pwebrtc
+
+import (
+	"encoding/hex"
+	"strings"
+
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/multiformats/go-multibase"
+	mh "github.com/multiformats/go-multihash"
+	"github.com/pion/webrtc/v3"
+)
+
+var mhToSdpHash = map[string]string{
+	"sha1":     "sha1",
+	"sha2-256": "sha-256",
+	"md5":      "md5",
+}
+
+var sdpHashToMh = map[string]string{
```

Comment:
This would probably be more performant as a function:
```go
func sdpHashToMultihash(s string) string {
     select s {
     case "sha1":
          return "sha1"
     ....
     }
}
```

The same applies to `mhToSdpHash`.